.golangci.yml

```diff
@@ -35,6 +35,7 @@ linters:
   - govet
   # linters default enabled by golangci-lint .
   - errcheck
+  - errorlint
```

Comment:
This linter is not enabled by default as per https://golangci-lint.run/usage/linters/#enabled-by-default.

Reply:
`errorlint` is disabled by default. I have seen both `fmt.Errorf("%s",err.Error)` and `fmt.Errorf("%v",err)` exist, so it is considered for better code specification. If the community feels it is not necessary, I will close the merge request

Reply:
I mean `errorlint` should be grouped into [other linters supported by golangci-lint](https://github.com/karmada-io/karmada/blob/2211ce63e4dd331dcda119f845dc8d3585794219/.golangci.yml#L43).